zebra/zebra_nhg.c

```diff
@@ -1122,6 +1123,16 @@ static void zebra_nhg_handle_install(struct nhg_hash_entry *nhe, bool install)
 			zebra_nhg_install_kernel(rb_node_dep->nhe);
 		}
 	}
+	/* update FIB flag of nexthop-group */
+	if (install) {
+		for (ALL_NEXTHOPS(nhe->nhg, nhop)) {
+			if (!CHECK_FLAG(nhop->flags, NEXTHOP_FLAG_ACTIVE))
+				continue;
+			if (CHECK_FLAG(nhop->flags, NEXTHOP_FLAG_RECURSIVE))
+				continue;
+			SET_FLAG(nhop->flags, NEXTHOP_FLAG_FIB);
```

Comment:
I want this flag *set* after we receive the correct response from the kernel / dplane that it has been properly installed.  This is not the correct place to make this change.    I thought we already did this though, so I am a bit confused here as well.

Reply:
Moved at the place where zebra_nhg_handle_install() is called.

This is for nexthop-groups. I have some issues with duplicate flag.
(see https://github.com/FRRouting/frr/pull/16028 )